File: anchor-reference-server/src/main/java/org/stellar/anchor/reference/event/processor/NoopEventProcessor.java
package org.stellar.anchor.reference.event.processor;

import org.stellar.anchor.api.event.AnchorEvent;

public class NoopEventProcessor implements SepAnchorEventProcessor {
  @Override
  public void onQuoteCreatedEvent(AnchorEvent event) {}

  @Override
  public void onTransactionCreated(AnchorEvent event) {}

  @Override
  public void onTransactionError(AnchorEvent event) {}

  @Override
  public void onTransactionStatusChanged(AnchorEvent event) {}

  @Override
  public void onCustomerUpdated(AnchorEvent event) {}
}

File: kotlin-reference-server/src/main/kotlin/org/stellar/reference/event/processor/NoOpEventProcessor.kt
package org.stellar.reference.event.processor

import org.stellar.reference.data.SendEventRequest

class NoOpEventProcessor : SepAnchorEventProcessor {
  override suspend fun onQuoteCreated(event: SendEventRequest) {}

  override suspend fun onTransactionCreated(event: SendEventRequest) {}

  override suspend fun onTransactionStatusChanged(event: SendEventRequest) {}

  override suspend fun onCustomerUpdated(event: SendEventRequest) {}
}
